view/frontend/web/js/internals/common.js

```diff
@@ -414,53 +414,6 @@ define(['jquery', 'algoliaInstantSearchLib', 'algoliaBase64', 'Magento_PageCache
             return hit;
         },
 
-        /** @deprecated This function should no longer be used and will be removed in a version 3.17 */
```

Comment:
Curious about why this needs to be removed in 3.16 rather than 3.17

Reply:
Yeah I originally thought to add a version number for removal, but I changed my mind while working on the Autocomplete feature. We actually previously marked this stuff deprecated but we did not provide a target release version for removal. However, because it obfuscates the code so much, I could not see any tangible benefit in leaving it in anymore. (It has not been relevant probably since version 3.8 and there's a cascade of related junk that I needed to excise in order to grok the CSS so I could try and clean it up.)